sam-local.yaml

```diff
@@ -13,7 +13,7 @@ Resources:
       CodeUri: build/exploded
       MemorySize: 320
       Policies: AWSLambdaBasicExecutionRole
-      Timeout: 15
+      Timeout: 29
```

Comment:
Why you use 29 instead of 30? Any reason for that?

Reply:
No specific reason